src/baldr/graphreader.cc

```diff
@@ -346,7 +346,8 @@ GraphReader::GraphReader(const boost::property_tree::ptree& pt,
     : tile_extract_(get_extract_instance(pt)), tile_dir_(pt.get<std::string>("tile_dir", "")),
       tile_getter_(std::move(tile_getter)),
       max_concurrent_users_(pt.get<size_t>("max_concurrent_reader_users", 1)),
-      tile_url_(pt.get<std::string>("tile_url", "")), cache_(TileCacheFactory::createTileCache(pt)) {
+      tile_url_(pt.get<std::string>("tile_url", "")), cache_(TileCacheFactory::createTileCache(pt)),
+      simulate_incidents_(pt.get<bool>("simulate_incidents", 0)) {
```

Comment:
this is temporary but makes it possible to completely disable this code path from config. we'll delete this once the presence of incident data begins to dictate whether or not incidents are returned